tests/dotnet-sos/sos_test.py

```diff
@@ -3,32 +3,62 @@
 
 import lldb
 import os
+import re
+import subprocess
 import sys
 
+def copy_libmscordaccore(ci, res):
+    ci.HandleCommand("finish", res)
+    assert(res.Succeeded())
+
+    ci.HandleCommand("p tmpdir", res)
+    assert(res.Succeeded())
+    tmpdir_match = re.search('"(.*)"', res.GetOutput())    
+    assert(tmpdir_match)
+    tmpdir_path = tmpdir_match.group(0)[1:-1]
```

Comment:
Assuming tmpdir_match returns a path, is this trying to get the path of the tmpdir? If that's the case it seems more readable to use os.path.basedir